src/PrestaShopBundle/Install/Upgrade.php

```diff
@@ -1175,13 +1183,13 @@ public static function migrateSettingsFile(Event $event = null)
                             'database_password' => _LEGACY_DB_PASSWD_,
                             'database_name' => _LEGACY_DB_NAME_,
                             'database_prefix' => _LEGACY_DB_PREFIX_,
-                            'database_engine' => _LEGACY_MYSQL_ENGINE_,
+                            'database_engine' => defined(_LEGACY_MYSQL_ENGINE_) ? _LEGACY_MYSQL_ENGINE_ : 'InnoDB',
                             'cookie_key' => _LEGACY_COOKIE_KEY_,
                             'cookie_iv' => _LEGACY_COOKIE_IV_,
                             'new_cookie_key' => _LEGACY_NEW_COOKIE_KEY_,
-                            'ps_caching' => _LEGACY_PS_CACHING_SYSTEM_,
-                            'ps_cache_enable' => _LEGACY_PS_CACHE_ENABLED_,
-                            'ps_creation_date' => _LEGACY_PS_CREATION_DATE_,
+                            'ps_caching' => defined(_LEGACY_PS_CACHING_SYSTEM_) ? _LEGACY_PS_CACHING_SYSTEM_ : 'CacheMemcache',
+                            'ps_cache_enable' => defined(_LEGACY_PS_CACHE_ENABLED_) ? _LEGACY_PS_CACHE_ENABLED_ : true,
```

Comment:
The PR has been updated, outdating my comment.
I disagree with this line, because this is not the cache you actually except. By setting this value as `true`, you will make PrestaShop activating the second cache (currently not known tfor real improvement on the loading time) and running the default cache class or 'CacheMemcache'.

I'm pretty sure this gonna break every shop which could not configure their cache properly, or haven't the extension required (like memcache).